### PR TITLE
Fixes the pneumatic cannon recipe not working

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Cheridan** [comment]
Is there something wrong with those other items used?

**Comment by Jordie0608** [comment]
You can't get both a manifold and valve onto the same tile, let alone a table too, it'd work if they were the item version and not machineries.

File: code/game/objects/items/weapons/pneumaticCannon.dm [hunk 1]
```diff
@@ -108,9 +108,7 @@
 				 /obj/item/weapon/wrench)
 	reqs = list(/obj/item/stack/sheet/metal = 4,
 				/obj/item/stack/packageWrap = 8,
-				/obj/item/pipe = 2,
-				/obj/machinery/atmospherics/binary/valve = 1,
-				/obj/machinery/atmospherics/pipe/manifold = 1,)
+				/obj/item/pipe = 2,)
```
Comment (Jordie0608) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: No comma at the end of the list

**Comment by Xhuis** [comment]
The item versions don't exist, as far as I can tell. They're just the /item/pipe item with different vars.

**Pull request merged**